:globe_with_meridians: :bento: add link to Okuna roadmap in useful links

stranljip opened the pull request:
Following the discussion on Slack from November 25th, I added a new link in the Useful links section of the app. The link is https://okuna.io/roadmap as suggested by @duichwer - so before merging this, a redirect has to be added to the webserver configuration. Also, I have not generated the changes to the i18n files as I do not have sufficient rights to upload something to Crowdin

Komposten commented:
@lifenautjoe This reminds me of something I thought of a while ago. Since we've had the Slack invite link in Useful Links fail at least once (maybe twice?), how about making also that into an okuna.io redirect (`okuna.io/slack`)? That would make it a lot easier to fix a broken link since it wouldn't require a new app version.

(Sorry @stranljip for hijacking your PR a bit here. :wink:)

stranljip commented:
Hi @Komposten - I hope I did everything right - I changed the target branch to develop and added the generated i18n files. Next time I will create the PR from the development branch in my fork, promised :-)
